packages/db/sequelize/foods/migrations/20230801154523-create-special-foods.js

```diff
@@ -0,0 +1,194 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) =>
+    queryInterface.sequelize.transaction(async (transaction) => {
+      // Create Special Foods Table and Special_Foods Steps Table and Special_Foods <-> Category <-> Foods Table)
+      await queryInterface.createTable(
+        'special_foods',
+        {
+          id: {
+            allowNull: false,
+            autoIncrement: true,
+            primaryKey: true,
+            type: Sequelize.BIGINT,
+          },
+          code: {
+            allowNull: false,
+            unique: true,
+            type: Sequelize.STRING(16),
+          },
+          name: {
+            allowNull: false,
+            type: Sequelize.STRING(128),
+          },
+          locale_id: {
+            allowNull: false,
+            type: Sequelize.STRING(16),
+          },
+          special_words: {
+            allowNull: false,
+            type: Sequelize.STRING(512),
+          },
+          synonyms: {
+            allowNull: false,
+            type: Sequelize.STRING(512),
+          },
+          created_at: {
+            allowNull: false,
+            defaultValue: Sequelize.NOW,
+            type: Sequelize.DATE,
+          },
+          updated_at: {
+            allowNull: false,
+            defaultValue: Sequelize.NOW,
+            type: Sequelize.DATE,
+          },
+          owner_id: {
```

Comment:
is `owner_id` field needed in this case? It's used for "record" level secururity and I don't we need that granular security for special foods?